package.json

```diff
@@ -101,9 +101,9 @@
     "@ckeditor/ckeditor5-paragraph": "^10.0.3",
     "@ckeditor/ckeditor5-table": "^11.0.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
-    "@fortawesome/free-brands-svg-icons": "^5.6.0",
-    "@fortawesome/free-regular-svg-icons": "^5.6.0",
-    "@fortawesome/free-solid-svg-icons": "^5.6.0",
+    "@fortawesome/free-brands-svg-icons": "^5.2.0",
```

Comment:
btw, why does it downgrade fontawesome?

Reply:
We could even upgrade it to 5.7

Reply:
No it does not exactly downgrade, it was like this before. After upgrading, it threw errors.

Reply:
But I'd love to update to 5.7